### PR TITLE
Move link to cellxgene data portal higher

### PR DESCRIPTION
**Pull request by lesliecodes**
image

### CODE REVIEW

**Review by MDunitz — APPROVED** [review]
lgtm

**Review by tihuan — APPROVED** [review]
LGTM as well!! Thanks so much, @lesliecodes 🙏 🎉 !!

File: docs/_config.yml [hunk 1]
```diff
@@ -13,10 +13,10 @@ nav:
     url: posts/install
   - title: Gallery
     url: posts/gallery
+  - title: Cellxgene data portal
```
Comment (seve) [inline docs/_config.yml:16]:
```suggestion
  - title: cellxgene data portal
```

File: docs/_site/index.html [hunk 2]
```diff
@@ -46,11 +46,11 @@
         
         
         
-        <a href="/cellxgene/posts/demo-data" class="btn">Demo datasets</a><br>
+        <a href="https://cellxgene.cziscience.com/" class="btn">Cellxgene data portal</a><br>
```
Comment (seve) [inline docs/_site/index.html:49]:
```suggestion
        <a href="https://cellxgene.cziscience.com/" class="btn">cellxgene data portal</a><br>
```

File: docs/_site/posts/annotations.html [hunk 3]
```diff
@@ -46,11 +46,11 @@
         
         
         
-        <a href="/cellxgene/posts/demo-data" class="btn">Demo datasets</a><br>
+        <a href="https://cellxgene.cziscience.com/" class="btn">Cellxgene data portal</a><br>
```
Comment (seve) [inline docs/_site/posts/annotations.html:49]:
```suggestion
        <a href="https://cellxgene.cziscience.com/" class="btn">cellxgene data portal</a><br>
```

File: docs/_site/posts/contact.html [hunk 4]
```diff
@@ -46,11 +46,11 @@
         
         
         
-        <a href="/cellxgene/posts/demo-data" class="btn">Demo datasets</a><br>
+        <a href="https://cellxgene.cziscience.com/" class="btn">Cellxgene data portal</a><br>
```
Comment (seve) [inline docs/_site/posts/contact.html:49]:
```suggestion
        <a href="https://cellxgene.cziscience.com/" class="btn">cellxgene data portal</a><br>
```

File: docs/_site/posts/gallery.html [hunk 5]
```diff
@@ -46,11 +46,11 @@
         
         
         
-        <a href="/cellxgene/posts/demo-data" class="btn">Demo datasets</a><br>
+        <a href="https://cellxgene.cziscience.com/" class="btn">Cellxgene data portal</a><br>
```
Comment (seve) [inline docs/_site/posts/gallery.html:49]:
```suggestion
        <a href="https://cellxgene.cziscience.com/" class="btn">cellxgene data portal</a><br>
```

File: docs/_site/posts/hosted.html [hunk 6]
```diff
@@ -46,11 +46,11 @@
         
         
         
-        <a href="/cellxgene/posts/demo-data" class="btn">Demo datasets</a><br>
+        <a href="https://cellxgene.cziscience.com/" class="btn">Cellxgene data portal</a><br>
```
Comment (seve) [inline docs/_site/posts/hosted.html:49]:
```suggestion
        <a href="https://cellxgene.cziscience.com/" class="btn">cellxgene data portal</a><br>
```

File: docs/_site/posts/install.html [hunk 7]
```diff
@@ -46,11 +46,11 @@
         
         
         
-        <a href="/cellxgene/posts/demo-data" class="btn">Demo datasets</a><br>
+        <a href="https://cellxgene.cziscience.com/" class="btn">Cellxgene data portal</a><br>
```
Comment (seve) [inline docs/_site/posts/install.html:49]:
```suggestion
        <a href="https://cellxgene.cziscience.com/" class="btn">cellxgene data portal</a><br>
```

File: docs/_site/posts/launch.html [hunk 8]
```diff
@@ -46,11 +46,11 @@
         
         
         
-        <a href="/cellxgene/posts/demo-data" class="btn">Demo datasets</a><br>
+        <a href="https://cellxgene.cziscience.com/" class="btn">Cellxgene data portal</a><br>
```
Comment (seve) [inline docs/_site/posts/launch.html:49]:
```suggestion
        <a href="https://cellxgene.cziscience.com/" class="btn">cellxgene data portal</a><br>
```

File: docs/_site/posts/methods.html [hunk 9]
```diff
@@ -46,11 +46,11 @@
         
         
         
-        <a href="/cellxgene/posts/demo-data" class="btn">Demo datasets</a><br>
+        <a href="https://cellxgene.cziscience.com/" class="btn">Cellxgene data portal</a><br>
```
Comment (seve) [inline docs/_site/posts/methods.html:49]:
```suggestion
        <a href="https://cellxgene.cziscience.com/" class="btn">cellxgene data portal</a><br>
```

File: docs/_site/posts/prepare.html [hunk 10]
```diff
@@ -46,11 +46,11 @@
         
         
         
-        <a href="/cellxgene/posts/demo-data" class="btn">Demo datasets</a><br>
+        <a href="https://cellxgene.cziscience.com/" class="btn">Cellxgene data portal</a><br>
```
Comment (seve) [inline docs/_site/posts/prepare.html:49]:
```suggestion
        <a href="https://cellxgene.cziscience.com/" class="btn">cellxgene data portal</a><br>
```

File: docs/_site/posts/roadmap.html [hunk 11]
```diff
@@ -46,11 +46,11 @@
         
         
         
-        <a href="/cellxgene/posts/demo-data" class="btn">Demo datasets</a><br>
+        <a href="https://cellxgene.cziscience.com/" class="btn">Cellxgene data portal</a><br>
```
Comment (seve) [inline docs/_site/posts/roadmap.html:49]:
```suggestion
        <a href="https://cellxgene.cziscience.com/" class="btn">cellxgene data portal</a><br>
```

File: docs/_site/posts/troubleshooting.html [hunk 12]
```diff
@@ -46,11 +46,11 @@
         
         
         
-        <a href="/cellxgene/posts/demo-data" class="btn">Demo datasets</a><br>
+        <a href="https://cellxgene.cziscience.com/" class="btn">Cellxgene data portal</a><br>
```
Comment (seve) [inline docs/_site/posts/troubleshooting.html:49]:
```suggestion
        <a href="https://cellxgene.cziscience.com/" class="btn">cellxgene data portal</a><br>
```

**Review by seve — COMMENTED** [review]
I think the preference is all lower case for `cellxgene`

**Comment by lesliecodes** [comment]
@seve I think since all the other links are capitalized, it'll look kinda weird if it were lowercase.

**Comment by seve** [comment]
> @seve I think since all the other links are capitalized, it'll look kinda weird if it were lowercase.

Truuue, LGTM then!

**Review by seve — APPROVED** [review]

**Pull request merged**